sdk/fabric/Azure.ResourceManager.Fabric/assets.json

```diff
@@ -0,0 +1,6 @@
+{
+  "AssetsRepo": "Azure/azure-sdk-assets",
+  "AssetsRepoPrefixPath": "net",
+  "TagPrefix": "net/<fabric/Azure.ResourceManager.Fabric",
+  "Tag": ""
```

Comment:
well you did not record your test cases and push the recordings to the recording repo.
Please follow the guidance here to record and push your recordings: https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/resourcemanager/Azure.ResourceManager/docs/TestGuide.md